pennylane/ops/qubit.py

```diff
@@ -1777,7 +1777,8 @@ class IsingXX(Operation):
 
     * Number of wires: 2
     * Number of parameters: 1
-    * Gradient recipe: :math:`\frac{d}{d \phi} \mathtt{XX}(\phi) = \frac{1}{2} \left[ \mathtt{XX}(\phi + \pi / 2) + \mathtt{XX}(\phi - \pi / 2) \right]`
+    * Gradient recipe: :math:`\frac{d}{d\theta}f(XX(\theta)) = \frac{1}{2}\left[f(XX(\theta +\pi/2)) - f(XX(\theta-\pi/2))\right]`
```

Comment:
Changed the `+` sign in between terms to `-`. Originally from [the Braket plugin](https://github.com/aws/amazon-braket-pennylane-plugin-python/blob/3a0a07af9e422e0fec2d89c0603a5e89ffe331e3/src/braket/pennylane_plugin/ops.py#L438) (will leave a note there).

Reply:
Oops, nice catch. Although, should we keep the parameter as `phi`, rather than `theta`, to match the equation above?

Reply:
Oh, that's a good idea! Updated